### PR TITLE
Write Servo-Pin Low when detaching

### PR DESCRIPTION
**Pull request by AnHardt**
regardles of the current state.
As sugested by @c-born in
https://github.com/MarlinFirmware/Marlin/issues/1885#issuecomment-92618240

### CODE REVIEW

File: Marlin/servo.cpp [hunk 1]
```diff
@@ -78,7 +78,7 @@ static inline void handle_interrupts(timer16_Sequence_t timer, volatile uint16_t
   if (Channel[timer] < 0)
     *TCNTn = 0; // channel set to -1 indicated that refresh interval completed so reset the timer
   else {
-    if (SERVO_INDEX(timer,Channel[timer]) < ServoCount && SERVO(timer,Channel[timer]).Pin.isActive)
+    if (SERVO_INDEX(timer,Channel[timer]) < ServoCount)
       digitalWrite( SERVO(timer,Channel[timer]).Pin.nbr,LOW); // pulse this channel low if activated
```
Comment (thinkyhead) [inline Marlin/servo.cpp:82]: The comment here can change too, then I'll merge it if it truly helps. I take it `.isActive` is just a software flag, and not testable in hardware.

**Comment by AnHardt** [comment]
@thinkyhead 
With my servos i could not reproduce the error. 
This PR is really only what is suggested in https://github.com/MarlinFirmware/Marlin/issues/1885#issuecomment-92618240.
As nothing happened on the servo-error-front, but issues piled up, i wanted to start some movement.
I really don't know if this helps. But with a PR it's much easier to test.

File: Marlin/servo.cpp (hunk 1 above)
Comment (c-born) [inline Marlin/servo.cpp:82]: Correct, it is just a flag to signify that the specific servo is active. The problem was that when using the servo deactivation delay, the servo could be made inactive after the delay period, but while the line was still high, and without this fix the line would then remain high, causing all sorts of bother. 
This fixes one specific problem, but the problem of continuous jitter i see reported must have another cause.

**Comment by thinkyhead** [comment]
> with a PR it's much easier to test

True. If you add the "PR Needs Testing" label to issues, I will avoid merging until there's more feedback.

**Pull request closed without merging**